spec/barbeque/executor_spec.rb

```diff
@@ -19,11 +19,22 @@
     end
 
     it 'performs a job' do
-      expect(test_job).to receive(:perform).with(*args)
+      expect(test_job).to receive(:perform).with('foo', 'bar')
```

Comment:
First of all, this line will work without change. So you intentionally introduce different coding style. I felt it's (of course consistent with L30 but) inconsistent with former lines using let.

But it's not strong opinion and personally both styles are okay...